### DISCUSSION TITLE
Reply-To header

### DISCUSSION THREAD

**Discussion by jdvalentine** [sabuhish/fastapi-mail#49] [Q&A]
Thanks for this excellent project which I'm enjoying using.

As a Python beginner, would supporting the Reply-To header be as simple as adding it in fastapi_mail/msg.py around line 86 along with an update to the MessageSchema? If so, I may be making my first GitHub pull request.

**Comment by sabuhish** [comment]
@jdvalentine  Thank you for your nice words :) I am glad that you like it.

Yes sure why not, we would welcome it definitely :) Make sure you tets it before sending PR. Waiting for your PR then.
